test/integration/tsconfig.json

```diff
@@ -1,22 +1,33 @@
 {
   "compilerOptions": {
     "allowSyntheticDefaultImports": true,
-    "lib": ["es2015.promise", "es2016", "es2017", "dom", "esnext.asynciterable"],
-    "module":"commonjs",
+    "outDir": "./.dist/",
+    "sourceMap": false,
+    "target": "es6",
+    "module": "commonjs",
     "moduleResolution": "node",
-    "declaration": true,
-    "declarationDir": "../dist/types",
+    "removeComments": true,
```

Comment:
this reduce bundle size?

Reply:
@orenyodfat hm... actually no. Bundle doesn't contain integration tests. I just copied this config from documentation. I can remove it. It is redundant and doesn't  do anything useful

Reply:
@orenyodfat removed because of redundant